src/_about/index.md

```diff
@@ -0,0 +1,53 @@
+---
+permalink: /about/
+title: About Us
+description: Learn about where Quilt came from, how its governance works, and how to get involved in structural changes.
+---
+
+Quilt is a mod-loader project - primarily aimed at Minecraft. It was founded by users and contributors from the Fabric 
+project, as a last-resort attempt to address the controversies and issues stemming from that project. Because Fabric 
+had a solid technological foundation with code that still did many things right, Quilt's founders decided that the 
+best option would be to take the standard open source approach to disagreements over management - they forked the 
+Fabric project into a new mod-loader project, with the following ideas in mind:
+
+* Allowing contributors to move fast and break things when appropriate, and allowing for experimental work
+* Building transparency into official community spaces and policies
+* Creating a community team that understands the importance of Quilt's values, and that fosters inclusivity, diversity 
+  and safety through an understanding hands-on moderation approach
+* Providing a governance structure that doesn't give anyone too much power, and avoids creating opportunities for 
+  staff members to operate without oversight
+* Providing ways to keep the community involved with important project decisions
+
+Quilt has grown quickly since its inception, with relatively broad support from the wider community. While some corners
+of the community have attempted to push back against its modern progressive values, Quilt continues to grow, attracting
+developers and users from all walks of life.
+
+# Governance
+
+Quilt takes an unusual approach to project governance, attempting to avoid giving any one person too much power over
+the project as a whole. This isn't a particularly easy task, especially when it comes to managing community spaces
+(and platforms like Discord that don't have the concept of group ownership). Quilt attempts to solve this problem by
+approaching it in a few specific ways:
+
+* Electing an Admin Board which exists to oversee the project's direction, manage permissions on GitHub, and break
```

Comment:
Perhaps it would be good to clarify that "elect" means from ourselves, not the community at large